pt-BR/action_controller_overview.md

```diff
@@ -1063,21 +1063,21 @@ Matching URLs will be marked as '[FILTERED]'.
 Rescue
 ------
 
-Most likely your application is going to contain bugs or otherwise throw an exception that needs to be handled. For example, if the user follows a link to a resource that no longer exists in the database, Active Record will throw the `ActiveRecord::RecordNotFound` exception.
+Muito provavelmente sua aplicação irá conter bugs ou enviar exceções que precisam ser tratadas. Por exemplo, se o usuário acessar um link que não possui uma fonte no banco de dados, o Active Record enviará `ActiveRecord::RecordNotFound` como exceção.
 
-Rails default exception handling displays a "500 Server Error" message for all exceptions. If the request was made locally, a nice traceback and some added information gets displayed so you can figure out what went wrong and deal with it. If the request was remote Rails will just display a simple "500 Server Error" message to the user, or a "404 Not Found" if there was a routing error or a record could not be found. Sometimes you might want to customize how these errors are caught and how they're displayed to the user. There are several levels of exception handling available in a Rails application:
+A exceção padrão do Rails apresenta a mensagem "500 Server Error" para todas as exceções. Se a requisição for feita localmente, um belo *traceback* e outras informações serão mostradas assim você pode verificar o que deu errado e tratar o problema. Se a requisição for remota o Rails apenas apresentará a mensagem "500 Server Error" para o usuário, ou um "404 Not Found" se houver um erro na rota ou o registro não puder ser encontrado. As vezes você pode querer customizar como esses erros são encontrados e como são apresentados ao usuário. Há diversos níveis de tratamento de excessões disponiveis em uma aplicação Rails:
```

Comment:
@akaninja aqui acontece a mesma ocorrência do outro PR. O que acha? As frases entre aspas ficam em itálico também?

Reply:
Eu acho positivo deixar em itálico porque ajuda a destacar que é uma expressão externa ao texto. Nesse caso em específico, não traduzimos as respostas padrão HTTP por ser um termo técnico.

Reply:
Fiquei em dúvida aqui.... 

Minha primeira reação seria italizar o termo e deixar entre aspas. Mas quando usamos as aspas, já damos a entender que é uma expressão estrangeira, então ficaria excessivo termos as duas estilizações. 
Talvez a gente tenha que avaliar caso a caso e não impor uma regra... 
Nesse caso específico, acho que poderia ficar só entre aspas ou italizar e tirar as aspas. Tendo a preferir deixar só aspas, porque nesse caso não é um termo técnico, é mais uma citação da mensagem... 

Ajudei? Ou só atrapalhei?